app/Http/Controllers/ChapterController.php

```diff
@@ -331,18 +331,15 @@ public function edit(Chapter $chapter)
      *
      * @return $this|bool|\Illuminate\Http\RedirectResponse
      */
-    public function update(Chapter $chapter)
+    public function update(Chapter $chapter, Request $request)
     {
         if (($redirect = $this->checkPermissions('EDIT_SHIP')) !== true) {
             return $redirect;
         }
 
-        $validator =
-            Validator::make($data = Request::all(), Chapter::$updateRules);
+        $this->validate($request, Chapter::$updateRules);
 
-        if ($validator->fails()) {
-            return Redirect::back()->withErrors($validator)->withInput();
-        }
+        $data = $request->all();
```

Comment:
The $this->validate() method handles returning the user to the form with the message.  This has been tested.

Now that we have a Request object, we can use the all() method instead of the Request::all() static, which was what was throwing the error